src/core/lint-hrefs-ids.js

```diff
@@ -0,0 +1,21 @@
+// Module core/lint-hrefs-ids
+// Warns about href's that do not have corresponding id's
+
+import { pub } from "core/pubsubhub";
+
+export const name = "core/lint-hrefs-ids";
+
+export function run(conf, doc, cb) {
```

Comment:
let's call this `export function lint(doc){}` instead. 

Then when importing it in  w3c/linter.js
```JS
import {lint as fragmentLinter} from "core/lint-hrefs-ids";
```

And let's maybe move "core/lint-hrefs-ids.js" to "core/linters/check-fragments.js" or something that makes it clear what the rule is.